src/component/admin/billing/BillingHistory/BillingHistory.tsx

```diff
@@ -0,0 +1,124 @@
+// TODO: finish this
+
+import {
+    Table,
+    SortableTableHeader,
+    TableBody,
+    TableCell,
+    TableRow,
+    TablePlaceholder,
+} from 'component/common/Table';
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { DateCell } from 'component/common/Table/cells/DateCell/DateCell';
+import { useMemo, VFC } from 'react';
+import { useGlobalFilter, useSortBy, useTable } from 'react-table';
+import { sortTypes } from 'utils/sortTypes';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { IconButton, styled, Typography } from '@mui/material';
+import FileDownload from '@mui/icons-material/FileDownload';
+
+interface IBillingHistoryProps {
+    data: Record<string, any>[];
+    isLoading?: boolean;
+}
+
+const columns = [
+    {
+        Header: 'Date',
+        accessor: 'createdAt',
+        Cell: DateCell,
+        sortType: 'date',
+    },
+    {
+        Header: 'Invoice nr.',
+        accessor: 'number',
+    },
+    {
+        Header: 'Payment method',
+        accessor: 'method',
+    },
+    {
+        Header: 'Amount',
+        accessor: 'amountFomratted', // TODO: typo
```

Comment:
We need to double check that the API actually includes the typo here aswell. This might just be a typo in the enterprise mock data, and not in the actual API response.